listView.php

```diff
@@ -73,23 +74,23 @@
        	$result = $stmt->execute($query_params); 
 }
 
-if (!empty($_POST['old_name']) && !empty($_POST['new_name'])){
-	//If the user requests to change the name of existing lists, this code is run - takes the old name, and new name, both strings as its input
+if (!empty($_POST['old_ID']) && !empty($_POST['new_name'])){
```

Comment:
good catch on finding this! I thought it was committed before but I guess it never went through.